dual-gauge-card.js

```diff
@@ -24,6 +24,16 @@ class DualGaugeCard extends HTMLElement {
     if (!this.config.max) {
       this.config.max = 100;
     }
+    
+    if (!this.config.fontsize) {
+      this.config.fontsize = 20;
+    }
+    if (!this.config.outer.fontsize) {
+      this.config.outer.fontsize = this.config.fontsize;
+    }
+    if (!this.config.inner.fontsize) {
+      this.config.inner.fontsize = this.config.fontsize;
+    }
```

Comment:
The default values for css-variables should be set via css. See my other suggestion below.
What should happen here is to use the global value for the inner and/or outer gauge, if available and no individual values are set.

```suggestion
    
    if (this.config.fontsize && !this.config.outer.fontsize) {
      this.config.outer.fontsize = this.config.fontsize;
    }
    if (this.config.fontsize && !this.config.inner.fontsize) {
      this.config.inner.fontsize = this.config.fontsize;
    }
```